apiserver/facades/client/modelconfig/modelconfig_test.go

```diff
@@ -79,7 +84,7 @@ func (s *modelconfigSuite) getAPI(c *gc.C) (*modelconfig.ModelConfigAPIV3, *gomo
 		}, nil,
 	).AnyTimes()
 
-	api, err := modelconfig.NewModelConfigAPI(s.backend, s.mockSecretBackendService, s.mockModelConfigService, &s.authorizer)
+	api, err := modelconfig.NewModelConfigAPI(coretesting.ModelTag.Id(), s.backend, s.mockSecretBackendService, s.mockModelConfigService, &s.authorizer)
```

Comment:
Please use `github.com/core/model/testing` to generate a new model id as per comments above. We don't need tags here and if we can reduce our dependency onto core testing the better.